docs/compass/03-01-security.md

```diff
@@ -16,46 +16,47 @@ The following diagram represents the architecture of the security in Compass:
 
 ### Tenant Mapping Handler
 
-It is an OathKeeper [hydrator](https://github.com/ory/docs/blob/525608c65694539384b785355d293bc0ad00da27/docs/oathkeeper/pipeline/mutator.md#hydrator) handler responsible for mapping authentication session data to tenant. It is built into Director itself, as it uses the same database. It is implemented as a separate endpoint, such as `/tenant-mapping`.
+It is an OathKeeper [hydrator](https://github.com/ory/docs/blob/525608c65694539384b785355d293bc0ad00da27/docs/oathkeeper/pipeline/mutator.md#hydrator) handler responsible for mapping authentication session data to tenant. It is built into the Director itself, as it uses the same database. It is implemented as a separate endpoint, such as `/tenant-mapping`.
 
 To unify the approach for mapping, we introduce `authorization_id`, widely used by multiple authentication flows.
 The `authorization_id` is equal to:
-- `client_id` in OAuth 2.0 authentication flow
-- `username` in Basic authentication flow
-- Common Name (CN) in Certificates authentication flow
+- `client_id` in the OAuth 2.0 authentication flow
+- `username` in the Basic authentication flow
+- Common Name (CN) in the Certificates authentication flow
+- Client ID in the one-time token authentication flow
 
-While generating one-time token, `client_id`/`client_secret` pair or basic authentication details for Runtime/Application/Integration System (using proper GraphQL mutation on Director), an entry in `system_auths` table in Director database is created. The `system_auths` table is used for tenant mapping.
+While generating one-time token, `client_id`/`client_secret` pair or basic authentication details for Runtime/Application/Integration System (using proper GraphQL mutation on the Director), an entry in `system_auths` table in the Director database is created. The `system_auths` table is used for tenant mapping.
 
-In certificates authentication flow, Tenant Mapping Handler puts fixed `scopes` into authentication session. The `scopes` are fixed in code, and they depends of type of the object (Application / Runtime / Integration System). In future we may introduce another database table for storing generic Application / Runtime / Integration System scopes.
+In certificates authentication flow, Tenant Mapping Handler puts fixed `scopes` into authentication session. The `scopes` are fixed in code, and they depend on the type of the object (Application/Runtime/Integration System). In the future we may introduce another database table for storing generic Application/Runtime/Integration System scopes.
```

Comment:
```suggestion
In certificates authentication flow, Tenant Mapping Handler puts fixed `scopes` into an authentication session. The `scopes` are fixed in code, and they depend on the object type (Application/Runtime/Integration System). In the future we are planning to introduce another database table for storing generic Application/Runtime/Integration System scopes.
```

Reply:
I'm not sure if writing about future plans is a good thing as there is always a chance it won't happen.

Reply:
I agree, it's just this document was a proposal and another TW is in the process of converting it into "regular" documentation in another PR.